src/__tests__/hooks.test.js

```diff
@@ -835,23 +829,20 @@ describe.skip('hooks', () => {
       return <div>{state && state.data}</div>;
     };
 
-    // Get the first child since the page is wrapped with the WithTracking component.
-    const page = await mount(<Page />).childAt(0);
-    await page.instance().executeAction(); // TODO: this probably doesn't work (how well does Enzyme support hooks?)
-    // TODO: maybe have to just call executeAction(); here
-    executeAction();
+    const page = await mount(<Page />);
+    await executeAction();
```

Comment:
Another option here (and maybe the more idiomatic option?) would be for `Page` to have a button or something that calls `executeAction` on click, and then using enzyme to simulate the click event, rather than calling it manually ourselves.